build/cqs.tsv

```diff
@@ -126,22 +118,22 @@ net.sf.geographiclib:GeographicLib-Java	1.49	compile
 net.sf.saxon:Saxon-HE	9.7.0-20	compile
 net.sf.trove4j:trove4j	3.0.3	compile
 org.antlr:antlr4-runtime	4.7.1	compile
-org.apache.accumulo:accumulo-core	1.8.1	compile
-org.apache.accumulo:accumulo-fate	1.8.1	compile
+org.apache.accumulo:accumulo-core	2.0.0	compile
 org.apache.arrow:arrow-format	0.16.0	compile
 org.apache.arrow:arrow-memory	0.16.0	compile
 org.apache.arrow:arrow-vector	0.16.0	compile
 org.apache.avro:avro	1.7.5	compile
 org.apache.camel:camel-core	2.24.1	compile
 org.apache.camel:camel-scala	2.24.1	compile
+org.apache.commons:commons-collections4	4.3	compile
```

Comment:
Hmm... 4.4 is approved: https://dev.eclipse.org/ipzilla/show_bug.cgi?id=21262.

I wonder if we should manage it up:).

Reply:
i'll give it a quick try and see if it builds ok

Reply:
builds ok so i went with it